### PR TITLE
[SDK][Agreement] Add agreement package to sdk.

### LINKED ISSUE #777: [CVAT][ORACLES][AGREEMENT] Separate agreement package from CVAT example
**Description**
Currently, the Inter-Rater Agreement package is a submodule of the CVAT example. It should be separated from the CVAT example and moved to the SDK. 

**Motivation**
Currently the agreement package is part of the CVAT example and thus cannot be reused in other labeling use-cases, limiting its utility.

**Implementation details** 
-

### PR DESCRIPTION
**Pull request by simulacrum6**
## Description

This PR adds a submodule to the the python sdk for calculating Interrater Agreement.

## Summary of changes

- Copied source code from CVAT example to python sdk.
- Copied tests from CVAT example to python sdk tests.
- Added dependencies `hypothesis` (dev) and `pyerf` and `numpy`.

## How test the changes

`make unit-test`

## Related issues

Closes #777.

## Operational checklist

- [x] All new functionality is covered by tests
- [ ] Any related documentation has been changed or added

### CODE REVIEW

**Comment by leric7** [comment]
I wonder if you have access to gitbook, but we'll need to update SDK docs.
In addition to, is this agreement module only limited to Python SDK? We don't want Typescript version of it?

**Comment by simulacrum6** [comment]
> I wonder if you have access to gitbook, but we'll need to update SDK docs. 
I can access the gitbook and make change requests. I suppose this is sufficient. Will make a change request and let you know once that is ready for review.


> In addition to, is this agreement module only limited to Python SDK? We don't want Typescript version of it?
Currently it is only planned to be part of the python SDK. It is currently also only an optional extra module for the python SDK, to keep its size small.
Maybe I should add checks and appropriate warnings, if people try to import functions from the agreement package and they have not installed the required extras? At the moment, it will throw an import error, saying that required libraries are not installed.

**Review by Dzeranov — APPROVED** [review]
lgtm, let's wait for approval from @leric7

**Pull request merged**
